doc/en/developer/source/index.rst

```diff
@@ -3,6 +3,9 @@ GeoServer Developer Manual
 
 Welcome to the GeoServer Developer Manual.  The manual is for those who want to help with the development process, including source code, software releasing, and other administrative work.
 
+Sadly we cannot create a Windows Installer at this time anymore.
```

Comment:
Can you reword in "At the time the GeoServer project does not have financial resources and man power to stand up a Windows build server (if you can help with this, please contact the developer list)."